Dockerfile

```
@@ -1,5 +1,6 @@
 # Dockerfile
 FROM rust:1.94.1-slim AS builder
+RUN apt-get update && apt-get install -y --no-install-recommends pkg-config && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)